src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,9 +1,70 @@
 <script setup>
+
+import { ref } from 'vue'
+
+const sentence = ref('')
+const N = ref(3)
+const combinations = ref([])
+
+const generateKeywords = () => {
+  // Reset combinations
+  combinations.value = []
+
+  const words = sentence.value.split(/\s+/)
+
+  for (let i = 1; i <= N.value; i++) {
+    const uniqueCombinations = new Set()
+
+    for (let j = 0; j <= words.length - i; j++) {
+      const combination = words.slice(j, j + i).join(' ')
+      uniqueCombinations.add(combination)
+    }
+
+    if (uniqueCombinations.size) {
+      combinations.value.push(Array.from(uniqueCombinations).join(', '))
+    }
+  }
+};
+
 </script>
+
 <template>
     <div class="ma-keywords-generator">
+
         <div class="ma-header">
-            <span>Keyword Generator </span>
+            <label class="text-black-500 text-3xl" for="allwords">Keyword Generator</label>
```

Comment:
We are not using tailwind for this task yet.

```suggestion
            <label for="allwords">Keyword Generator</label>
```